frontend/src/components/common/Dropdown/DropdownListItem/DropdownListItem.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="dropdown__item item">
+    <div class="item__heading">
+      <h3 class="item__title">{{ title }}</h3>
+      <span class="item__time">{{ time }}</span>
+    </div>
+    <p class="item__description">{{ sliced }}</p>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    title: {
+      type: String,
+    },
+    time: {
+      type: String,
+    },
+    description: {
+      type: String,
+    },
+  },
+  setup(props) {
+    let sliced = props.description.slice(0, 52);
```

Comment:
This can be achieved by using `overflow-ellipsis` on the paragraph with the text:
https://tailwindcss.com/docs/text-overflow